#include <exception>

void print_exception(const std::exception& e, int level = 0);
